leaf_elements.go

```diff
@@ -172,6 +172,7 @@ var ofxLeafElements = []string{
 	"IDSCOPE",
 	"INCBAL",
 	"INCIMAGES",
+	"INCLUDE",
```

Comment:
I wonder what my regex at the top of this file is missing that it left out `INCLUDE`... this makes me nervous that there may be other similar fields also missing.

Reply:
It looks like the `INCLUDE` one is a total oddball case in those .dtd files. It's got this extra semicolon before the right-paren.... and it's the only one like that. I'm not at all familiar with .dtd files but it seems pretty strange this is the only one with special syntax. 🤷‍♂ 

I can update the regex so it also picks up the `INCLUDE` one too, if you like